packages/esm-ward-app/src/config-schema.ts

```diff
@@ -6,16 +6,17 @@ const defaultWardPatientCard: WardPatientCardDefinition = {
   rows: [
     {
       rowType: 'header',
-      elements: ['bed-number', 'patient-name', 'patient-age', 'patient-address'],
+      elements: ['bed-number', 'patient-name', 'patient-age', 'patient-address', 'patient-identifier'],
     },
   ],
   appliedTo: null,
 };
 
 const defaultPatientAddressFields: Array<keyof PersonAddress> = ['cityVillage', 'country'];
-
+const defaultIdentifierTypeUuid = '05a29f94-c0ed-11e2-94be-8c13b969e334';//OpenMrs ID
 export const defaultPatientCardElementConfig: PatientCardElementConfig = {
   addressFields: defaultPatientAddressFields,
+  identifierTypeUuid: defaultIdentifierTypeUuid,
```

Comment:
I'd just make this set to `null` by default, and have the element use the preferred patient identifier by default. (See comment below)

Reply:
Agreed.  Without a specific identifierType configured, it should not default to restrict to a particular type